elapsed_time_block.py

```diff
@@ -71,121 +73,52 @@ def _get_timedelta(self, signal):
-                        delta['seconds'] = seconds % 60
-                elif self.units().hours(signal):
-                    if not self._more_significant_selected('hours', signal):
-                        # seconds and hours
-                        delta['hours'] = int(hours)
-                        delta['seconds'] = seconds % (delta['hours'] * 60**2)
-                    else:
-                        # seconds and hours and days
-                        delta['days'] = int(days)
-                        delta['hours'] = int(hours % (delta['days'] * 24))
-                        delta['seconds'] = seconds % (delta['hours'] * 60**2)
-                elif self.units().days(signal):
-                    # seconds and days
-                    delta['days'] = int(days)
-                    delta['seconds'] = seconds % (delta['days'] * 60**2 * 24)
-            elif self.units().minutes(signal):
-                if not self._more_significant_selected('minutes', signal):
-                    # minutes only
-                    delta['minutes'] = minutes
-                elif self.units().hours(signal):
-                    if not self._more_significant_selected('hours', signal):
-                        # minutes and hours
-                        delta['hours'] = int(hours)
-                        delta['minutes'] = minutes % (delta['hours'] * 60)
-                    else:
-                        # minutes and hours and days
-                        delta['days'] = int(days)
-                        delta['hours'] = int(hours % (delta['days'] * 24))
-                        delta['minutes'] = minutes % (delta['hours'] * 60)
-                elif self.units().days(signal):
-                    # minutes and days
-                    delta['days'] = int(days)
-                    delta['minutes'] = minutes % (delta['days'] * 60 * 24)
-            elif self.units().hours(signal):
-                if not self._more_significant_selected('hours', signal):
-                    # hours only
-                    delta['hours'] = hours
-                else:
-                    # hours and days
-                    delta['days'] = int(days)
-                    delta['hours'] = hours % (delta['days'] * 24)
-            elif self.units().days(signal):
-                # days only
-                delta['days'] = days
-        return delta
+        return seconds
+
+    def _format_seconds_diff(self, seconds, signal):
+        """ Returns the number of seconds in terms of `units` using `signal`"""
+        days_enabled = self.units().days(signal)
+        hours_enabled = self.units().hours(signal)
+        mins_enabled = self.units().minutes(signal)
+        secs_enabled = self.units().seconds(signal)
+        least_significant = "seconds"
+        least_significant_mult = 1
+
+        if not any([days_enabled, hours_enabled, mins_enabled, secs_enabled]):
+            # Treat nothing enabled as all enabled
```

Comment:
This was me misunderstanding the requirements of the original code. I would probably change this to raise an error or just return an empty dict at this point.